src/hooks/useSyncTokenSymbolToUrl.ts

```diff
@@ -44,7 +44,8 @@ export default function useSyncTokenSymbolToUrl(
   const { pathname } = useLocation()
   const defaultTokens = useAllTokens()
```

Comment:
Variable naming `const defaultTokens = useAllTokens()`???

Reply:
Ok, i renamed it to `allTokens`